### ISSUE TITLE
Tests are failing in clean chroot

### ISSUE THREAD

**Issue by orhun** [woodruffw/kbs2#124]
Hey,
Recently I added tests to the [kbs2](https://aur.archlinux.org/packages/kbs2/) package and tried to build it in a clean chroot. Unfortunately it fails due to [test_find_default_config_dir](https://github.com/woodruffw/kbs2/blob/eff3964ff7ee75da04cf2b0a87c1e1d84da1277c/src/kbs2/config.rs#L465) function.

```sh
failures:

---- kbs2::config::tests::test_find_default_config_dir stdout ----
thread 'kbs2::config::tests::test_find_default_config_dir' panicked at 'assertion failed: parent.exists()', src/kbs2/config.rs:473:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    kbs2::config::tests::test_find_default_config_dir

test result: FAILED. 26 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out
```

It's because home directory points to `/root/` so `/root/.config/` doesn't exist. Tests are passing when I remove the last 2 lines from the function:

```rust
    #[test]
    fn test_find_default_config_dir() {
        let dir = find_default_config_dir().unwrap();
        // NOTE: We can't check whether the main config dir exists since we create it if it
        // doesn't; instead, we just check that it isn't something weird like a regular file.
        assert!(!dir.is_file());

        let parent = dir.parent().unwrap();

        //assert!(parent.exists());
        //assert!(parent.is_dir());
    }
```

What can we do about this?

**Comment by woodruffw** [comment]
Yeah, this test never made much sense in the first place: we always create the default config dir (and all parents) if it doesn't exist, so we shouldn't be testing for their existence. 

I'll remove that test entirely in a bit and cut a patch.

**Comment by woodruffw** [comment]
Okay, fixed and cut with 0.2.5. Should be released once the CI completes.
